pkgs/by-name/se/sessionizer/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  pkgs,
+  buildGoModule,
+}:
+buildGoModule {
+  pname = "sessionizer";
+  version = "0.1.4";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "salfel";
+    repo = "sessionizer";
+    rev = "v0.1.4";
+    sha256 = "sha256-JZaJPxUEkUcNKB5EvoBuoMpiSZuB+q3hyl+tYjcI/ag=";
+  };
+
+  goPackagePath = "github.com/salfel/sessionizer";
+
+  vendorHash = "sha256-04uCddd3+5YlAATkrNciHaem9lX/2P8AHDfRDH0ut7c=";
+
+  proxyVendor = true;
+
+  buildInputs = [ pkgs.fzf ];
```

```suggestion
  nativeBuildInputs = [ makeBinaryWrapper ];
  
  postFixup = ''
    wrapProgram $out/bin/sessionizer --prefix PATH : ${lib.makeBinPath [ fzf ]}
  '';
```

`fzf` should come from input directly as well.